python/dask_cudf/dask_cudf/backends.py

```diff
@@ -296,12 +297,17 @@ def is_categorical_dtype_cudf(obj):
     return cudf.api.types.is_categorical_dtype(obj)
 
 
+@grouper_dispatch.register((cudf.Series, cudf.DataFrame))
+def get_grouper_cudf(obj):
+    return cudf.core.groupby.Grouper
+
+
 try:
-    from dask.dataframe.dispatch import grouper_dispatch
+    from dask.dataframe.dispatch import pyarrow_schema_dispatch
```

Comment:
Do we need to bump the minimum Dask version because of this?

Reply:
Not a pressing issue to bump min requirement, since it is in `try/except` block the current code should just work for old and new version of dask.